src/views/Login.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from "react";
+import { Form, Image, Alert } from "react-bootstrap";
+import "../assets/styles/containers/loginPage.scss";
+import { connect } from "react-redux";
+import {
+  fetchUsers,
+  validateInput,
+  loginUsersSuccess,
+} from "../redux/actions/index";
+import { Redirect } from "react-router-dom";
+import Textbox from "../components/Textbox";
+import Button from "../components/Button";
+import Line from "../components/line";
+import { postThunk } from "../redux/thunk/index";
+import { withRouter } from "react-router";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: { email: "", password: "" },
+      error: " ",
+      loggedIn: false,
+    };
+  }
+  render() {
+    const { validateInput } = this.props;
+    const emailValidation = this.props.validations.email;
+    const passwordValidation = this.props.validations.password;
+    const { error } = this.state;
+    return (
+      <div className="login-page">
+        <Form>
+          <h1>Sign in</h1>
+          <span>
+            <p className="text-danger" id="loginError">
+              {error ? error : " "}
+            </p>
+          </span>
+          <p>Login with so social media</p>
+          <div className="Google-login-btn">
```

Comment:
Please refactor this file by making some of the parts of its components.

Reply:
@kagabof This will be done when implementing the social login